src/services/conversionRates/getConversionRatesService.js

```diff
@@ -283,10 +285,13 @@ const getConversionRates = async (app, requestedDate, requestedSymbol = 'ETH') =
 
   const tokens = app.get('activeTokenWhitelist');
```

Comment:
use `tokenWhiteList` instead of `activeTokenWhiteList` please, that was a mistake.

Reply:
done